src/services/ExplorerService/fetcher.ts

```diff
@@ -1138,3 +1161,13 @@ type SubmitTokenInfoParams = {
   uan?: string
   token?: string
 }
+
+export interface RGBTransaction {
+  txHash: string
+  blockId: number
+  blockNumber: number
+  blockTimestamp: number
+  leapDirection: string
+  rgbCellChanges: number
+  tgbTxid: string
```

Comment:
It's better to handle the typo compatibility in advance, e.g.

1. define both `tgbTxid` and `rgbTxid`
2. Use it as `rgbTxid ?? tgbTxid`
3. add a `FIXME` comment

By doing so, the API can be fixed anytime.

Reply:
> It's better to handle the typo compatibility in advance, e.g.
> 
> 1. define both `tgbTxid` and `rgbTxid`
> 2. Use it as `rgbTxid ?? tgbTxid`
> 3. add a `FIXME` comment
> 
> By doing so, the API can be fixed anytime.

Code missed in the commit.

Resubmitted in 6874bdbb526f39db0a8f28514aa27adca2ae9c6d